csharp/tests/Integration/GetAndSet.cs

```diff
@@ -2,19 +2,12 @@
  * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
  */
 
-namespace tests;
+namespace tests.Integration;
 
 using Glide;
 
-// TODO - need to start a new redis server for each test?
-public class AsyncClientTests
+public class GetAndSet
```

Comment:
why is this a class? what's the intended separation between test classes?

Reply:
No separation, I just renamed it. I assume we won't put all tests into a single class.